kokoro/scripts/test/go_test.sh

```diff
@@ -213,7 +208,11 @@ if [[ "${SHORT:-false}" == "true" ]]; then
 fi
 
 TEST_UNDECLARED_OUTPUTS_DIR="${LOGS_DIR}" \
-  go test -v "${TEST_SUITE_NAME}.go" \
+  gotestsum \
+  "${GOTESTSUM_ARGS:-}" \
+  --packages=./"${TEST_SUITE_NAME}.go" \
+  --junitfile "${LOGS_DIR}/sponge_log.xml" \
+  -- -v  \
```

Comment:
I think we shouldd use gotestsum flag `--format=standard-verbose` to replace this use of `-v` here. Looking at a passing log, I don't think this `-v` is doing anything for us anymore, and the output it used to be printing has been lost: https://btx-internal.corp.google.com/invocations/decb3aed-be92-4f20-9615-eef393aff7ad/log

Reply:
Done.